<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output Location</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="saveWithInputButton">
          <property name="toolTip">
           <string>Save batch output datasets in the same directories as the input datasets</string>
          </property>
          <property name="text">
           <string>Save with Input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="saveToDirButton">
          <property name="toolTip">
           <string>Save all batch output datasets to the same directory</string>
          </property>
          <property name="text">
           <string>Save to Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="outputDirEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Batch output dataset directory</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outputDirChooseButton">
            <property name="toolTip">
             <string>Select the batch output dataset directory</string>
            </property>
            <property name="text">
             <string>Choose...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Output Suffix</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="outputSuffixEdit">
          <property name="toolTip">
           <string>Batch output dataset naming convention: input_name + suffix</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Export Format</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QComboBox" name="exportFormatCombo">
          <property name="toolTip">
           <string>Select the export dataset storage format</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="exportAllButton">
          <property name="toolTip">
           <string>Compute and store all batch output datasets</string>
          </property>
          <property name="text">
           <string>Export All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAllButton">
          <property name="toolTip">
           <string>Delete all previously computed batch output datasets</string>
          </property>
          <property name="text">
           <string>Delete All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
